Library/Homebrew/extend/ENV/super.rb

```diff
@@ -199,10 +199,15 @@ def homebrew_extra_library_paths
 
   sig { returns(T.nilable(PATH)) }
   def determine_library_paths
-    paths = [
-      keg_only_deps.map(&:opt_lib),
-      HOMEBREW_PREFIX/"lib",
-    ]
+    paths = []
+    if compiler.match?(GNU_GCC_REGEXP)
+      f = gcc_version_formula(compiler.to_s)
+      paths << f.opt_lib/"gcc/#{f.version.major}" if f.any_version_installed?
```

Comment:
Minor style nit.
```suggestion
      paths << f.opt_lib/"gcc"/f.version.major if f.any_version_installed?
```
We don't need interpolation here. (Probably doesn't make a huge difference, but I find this more readable.)